test/fixtures.rb

```diff
@@ -86,6 +86,9 @@ class Movie < ActiveRecord::Base
   has_and_belongs_to_many :genres
   has_and_belongs_to_many :actors
   has_and_belongs_to_many :writers
+
+  scope :classics, -> { where('year <= ?', 1980) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.